packages/next-tinacms-dos/src/handlers.ts

```diff
@@ -109,6 +109,9 @@ async function uploadMedia(
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
   const filePath = req.file.path
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const fileType = req.file.mimetype
```

Comment:
```suggestion
  const fileType = req.file?.mimetype
```

Does this need a `?`